### PR TITLE
Feature book repository

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Adding book repository with method to find a book by author
## Trello Card
https://trello.com/c/0i7JwAxP
## Test
image

### CODE REVIEW

File: src/main/java/com/wolox/training/repositories/BookRepository.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import com.wolox.training.models.Book;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    public Optional<Book> findByAuthor(String author);
```
Comment (jdhiguita) [inline src/main/java/com/wolox/training/repositories/BookRepository.java:16]: It's a great time to add java docs to our project. It is a very simple way to start documenting your application with comments that explain the functionality of certain methods and classes (Java Docs).

A valid implementation for this method would be to add a short description of what the method is expected to do, add the @param to indicate what it expects from input and @return to indicate its output.

**Review by torsello — APPROVED** [review]
LGTM!

**Review by jdhiguita — COMMENTED** [review]
Great job, some comments

File: src/main/java/com/wolox/training/repositories/BookRepository.java (hunk 1 above)
Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/repositories/BookRepository.java:16]: Thanks Juancho! I pushed the change!

**Review by jdhiguita — APPROVED** [review]

**Pull request merged**